cpp/tests/copying/scatter_list_tests.cpp

```diff
@@ -35,7 +35,8 @@ class TypedScatterListsTest : public cudf::test::BaseFixture {
 using FixedWidthTypes = cudf::test::Concat<cudf::test::IntegralTypes,
                                            cudf::test::FloatingPointTypes,
                                            cudf::test::DurationTypes,
-                                           cudf::test::TimestampTypes>;
+                                           cudf::test::TimestampTypes,
+                                           cudf::test::FixedPointTypes>;
 TYPED_TEST_CASE(TypedScatterListsTest, FixedWidthTypes);
```

Comment:
```suggestion
TYPED_TEST_CASE(TypedScatterListsTest, cudf::test::FixedWidthTypes);
```

We can remove the local definition of `FixedWidthTypes`.

Reply:
Done.